Gemfile

```diff
@@ -26,7 +26,7 @@ gem 'react-rails', '~> 1.7.1'
 gem 'mailgun_rails', '~> 0.8.0'
 gem 'opencpu', '~> 0.10.0'
 gem 'active_interaction', '~> 3.0'
-gem 'jsonb_accessor', '~> 0.3.2'
+gem 'jsonb_accessor', github: 'devmynd/jsonb_accessor', branch: '0.x-versions'
```

Comment:
Correct, de oude versie is niet compatible met de nieuwe ActiveRecord connection adapter.